modules/navier_stokes/test/tests/finite_volume/ins/jeffery-hamel/wedge_dirichlet_vol_PA.i

```diff
@@ -0,0 +1,192 @@
+mu=.01
+rho=1
+
+# This input file tests whether we can converge to the semi-analytical
+# solution for flow in a 2D wedge.
+[GlobalParams]
+  velocity_interp_method = 'rc'
+  advected_interp_method = 'average'
+  rhie_chow_user_object = 'rc'
+[]
+
+[Mesh]
+  [file]
+    type = FileMeshGenerator
+    # file = wedge_4x6.e
+    file = wedge_8x12.e
+    # file = wedge_16x24.e
+    # file = wedge_32x48.e
+    # file = wedge_64x96.e
+  []
+  [./corner_node]
+    # Pin is on the centerline of the channel on the left-hand side of
+    # the domain at r=1.  If you change the domain, you will need to
+    # update this pin location for the pressure exact solution to
+    # work.
+    type = ExtraNodesetGenerator
+    new_boundary = pinned_node
+    coord = '1 0'
+    input = file
+  [../]
+[]
+
+[UserObjects]
+  [rc]
+    type = INSFVRhieChowInterpolator
+    u = vel_x
+    v = vel_y
+    pressure = pressure
+  []
+[]
+
+[Variables]
+  [vel_x]
+    type = INSFVVelocityVariable
+  []
+  [vel_y]
+    type = INSFVVelocityVariable
+  []
+  [pressure]
+    type = INSFVPressureVariable
+  []
+[]
+
+[AuxVariables]
+  [U]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
+  []
+[]
+
+[AuxKernels]
+  [mag]
+    type = VectorMagnitudeAux
+    variable = U
+    x = vel_x
+    y = vel_y
+  []
+[]
+
+[FVKernels]
+  [mass]
+    type = INSFVMassAdvection
+    variable = pressure
+    rho = ${rho}
+  []
+
+  [u_advection]
+    type = INSFVMomentumAdvection
+    variable = vel_x
+    rho = ${rho}
+    momentum_component = 'x'
+  []
+
+  [u_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = vel_x
+    mu = 'mu'
+    momentum_component = 'x'
+  []
+
+  [u_pressure]
+    type = INSFVMomentumPressure
+    variable = vel_x
+    momentum_component = 'x'
+    pressure = pressure
+  []
+
+  [v_advection]
+    type = INSFVMomentumAdvection
+    variable = vel_y
+    rho = ${rho}
+    momentum_component = 'y'
+  []
+
+  [v_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = vel_y
+    mu = 'mu'
+    momentum_component = 'y'
+  []
+
+  [v_pressure]
+    type = INSFVMomentumPressure
+    variable = vel_y
+    momentum_component = 'y'
+    pressure = pressure
+  []
+[]
+
+[FVBCs]
+
+  [no_slip_x]
+    type = INSFVNoSlipWallBC
+    variable = vel_x
+    boundary = 'top_wall bottom_wall'
+    function = 0
+  []
+
+  [no_slip_y]
+    type = INSFVNoSlipWallBC
+    variable = vel_y
+    boundary = 'top_wall bottom_wall'
+    function = 0
+  []
+
+  [inlet_x]
+    type = INSFVInletVelocityBC
+    variable = vel_x
+    boundary = 'inlet'
+    function = 1
+  []
+
+  [inlet_y]
+    type = INSFVInletVelocityBC
+    variable = vel_y
+    boundary = 'inlet'
+    function = 0
+  []
+
+  [outlet_p]
+    type = INSFVOutletPressureBC
+    boundary = 'outlet'
+    variable = pressure
+    function = 0
+  []
```

Comment:
So in order to match the finite element results, we need to change to match the finite element boundary conditions, which are:
```
  [./vel_x_inlet]
    type = FunctionDirichletBC
    variable = vel_x
    boundary = 'inlet outlet'
    function = 'vel_x_exact'
  [../]
  [./vel_y_inlet]
    type = FunctionDirichletBC
    variable = vel_y
    boundary = 'inlet outlet'
    function = 'vel_y_exact'
  [../]
  [./pressure_pin]
    type = DirichletBC
    variable = p
    boundary = 'pinned_node'
    value = 0
  [../]
```
So what this should change to is:
```
 [inlet_x]
    type = INSFVInletVelocityBC
    variable = vel_x
    boundary = 'inlet outlet'
    function = 'vel_x_exact'
  []

  [inlet_y]
    type = INSFVInletVelocityBC
    variable = vel_y
    boundary = 'inlet outlet'
    function = 'vel_y_exact'
  []
```
and then you'll need to add the wedge functions in a `Functions` block:
```
[Functions]
  [vel_x_exact]
    type = WedgeFunction
    var_num = 0
    mu = 1
    rho = 1
  []
  [vel_y_exact]
    type = WedgeFunction
    var_num = 1
    mu = 1
    rho = 1
  []
[]
```
And we'll need to add a scalar Lagrange multiplier constraint like we have for the finite volume lid driven cavity input